Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM amazoncorretto:11-alpine-jdk
```

Comment:
BTW, I don't see anywhere container for testing :)

Reply:
Added in #6 PR